user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_triggering_user_defined_api.md

```diff
@@ -2,24 +2,29 @@
 uid: SLNetClientTest_triggering_api
 ---
 
-# Triggering a user-defined API
+# Triggering and managing user-defined APIs
+
+> [!NOTE]
+> This feature is available from DataMiner 10.3.6/10.4.0 onwards. In DataMiner 10.3.5, it is available in preview.
 
 > [!WARNING]
-> This feature is in preview and is not fully released yet. For now, it should only be used on a staging platform. It should not be used in a production environment.
+> Always be extremely careful when using the SLNetClientTest tool, as it can have far-reaching consequences on the functionality of your DataMiner System.
 
-With the SLNetClientTest tool, you can trigger a user-defined API in order to test it. This especially handy when you are developing an API, as it allows you to quickly and easily trigger it and get additional information such as the response time along with the result.
+## Triggering a user-defined API
+
+With the SLNetClientTest tool, you can trigger a user-defined API in order to test it. This will bypass the endpoint DxM and go directly to the API manager in SLNet, which can be useful to efficiently test and verify API scripts without the need to send an HTTP request. This is especially handy when you are developing an API, as it allows you to quickly and easily trigger it and get additional information such as the response time along with the result. It can also be used to help detect whether an API script is faulty or if something else is the cause when problems occur with API triggers.
 
 1. [Connect to the DMA using the SLNetClientTest tool](xref:Connecting_to_a_DMA_with_the_SLNetClientTest_tool).
 
-1. Select *Advanced* > *Apps* > *User-defined APIs*. <!-- RN 35996 -->
+1. Select *Advanced* > *Apps* > *User-Defined APIs*. <!-- RN 35996 -->
 
 1. Select the *Trigger* tab.
 
    ![Client Test Tool Trigger Screenshot](~/user-guide/images/UDAPIS_ClientTestToolTrigger.png)
 
 1. In the *API Definition* box, select the API you want to trigger.
```

Comment:
The API Definition box doesn't exist anymore?

Reply:
Indeed, this should be *Route*

Reply:
Updated!